bnd-maven-plugin-parent/LICENSE

```diff
@@ -0,0 +1,177 @@
+
```

Comment:
We already have a LICENSE file in the root of the REPO. Do we need another one here?